internal/pkg/agent/cmd/run.go

```diff
@@ -622,6 +622,8 @@ func startOldAgentConfigCleaner(ctx context.Context, log *logp.Logger) {
 		return
 	}
 
+	log.Warnf("Everything is magic")
```

Comment:
![](https://c.tenor.com/NoH1i8kN8iAAAAAC/but-why-wtf.gif)

Reply:
@aleksmaus Yeah, just wanted to get the ci to run :)